### DISCUSSION TITLE
Windows Keeps Looking for a Network Connection - Opens New Browser or Tab

### DISCUSSION THREAD

**Discussion by themak1985** [utmapp/UTM#2590] [General]
**Describe the issue**
So Windows 10 ARM Edition	Windows 10 Pro (21354.1) on UTM is super buggy - see #2569 - but I am encountering a weird issue. Windows is opening msftconnecttest.com/redirect via edge (or any browser) which is redirecting to https://www.msn.com/?ocid=wispr&pc=u477 every minute or so. Not sure how to stop it.

**Configuration**
* UTM Version: 2.1.1
* OS Version:  Big Sur
* Intel or Apple Silicon? Silicon

**Comment by PDVJAM** [comment]
Exactly the same issue.

**Comment by conath** [comment]
I believe I have the solution for you. There is a registry key you can change to disable this connect test:

Go to `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\NlaSvc\Parameters\Internet` and set `EnableActiveProbing` to `0`.

This worked in my Windows 10 VM in UTM. Source: https://networkproguide.com/fix-connect-attempts-to-www-msftconnecttest-com-windows-server-2016/

.reg file if you need it: 
[disable-ms-connect-test-popup.reg.zip](https://github.com/utmapp/UTM/files/6598432/disable-ms-connect-test-popup.reg.zip)

**Reply by conath** [reply]
Note that this will mean Windows always shows as No Internet even though you are, in fact, online. You can poke around with the other registry values in that path, maybe it's possible to get the Internet Status showing online correctly. I have not tried that.

**Reply by ggruber4711** [reply]
Thanks for this, saved my day :-)

**Comment by conath** [comment]
This will be fixed in UTM update 2.2.0 or later, as it introduces Shared & Bridged networking. The underlying issue with emulated networking will not be fixed by the 2.2 update, however switching to Shared or Bridged Mode will workaround the problem entirely. I have personally tested it.

**Reply by jlfreund** [reply]
I have the same problem.  When I come back to my computer, Chrome has about 20 tabs open to this URL:

https://www.msn.com/?ocid=wispr&pc=u477&AR=2

This seems to have started happening recently (sometime in September, 2025).

I did not experience any issues with "No Internet" being displayed anywhere.  My internet works fine, which is why the MSN landing page is loading fine.

I don't understand what is the connection between: "Shared & Bridged networking" and this MSN spam.  I do not have any VPN service running, and the explanation feels a little hollow, as it seems odd that some kind of low level network behavior would invoke an MSN landing page.  My guess is that this is a bug (or deliberate) problem with some Microsoft pre-installed application (News? Accounts?) to attract attention to an MSN landing page.
